RadixWallet/Clients/AccountLockersClient/AccountLockersClient+Live.swift

```diff
@@ -0,0 +1,210 @@
+import Foundation
+
+public typealias ClaimsPerAccount = [AccountAddress: [AccountLockerClaims]]
+
+extension AccountLockersClient {
+	public static let liveValue: Self = .live()
+
+	public static func live() -> AccountLockersClient {
+		@Dependency(\.authorizedDappsClient) var authorizedDappsClient
+		@Dependency(\.accountsClient) var accountsClient
+		@Dependency(\.onLedgerEntitiesClient) var onLedgerEntitiesClient
+		@Dependency(\.gatewayAPIClient) var gatewayAPIClient
+		@Dependency(\.cacheClient) var cacheClient
+
+		let claimsPerAccountSubject = AsyncCurrentValueSubject<ClaimsPerAccount>([:])
+
+		let timerInterval: DispatchTimeInterval = .seconds(600) // 10 minutes
+
+		@Sendable
+		func startMonitoring() async throws {
+			let dappValues = await authorizedDappsClient.authorizedDappValues()
+			let accountValues = await accountsClient.accountsOnCurrentNetwork()
```

Comment:
yeah, this will trigger on any dapp or account change. For dApps it make sense to trigger always (since linked accounts or deposit hidden/visible must trigger new check). 

For accounts, I will update it so that we only care when the ids of the account change (and not the account details as the name).